src/csharp/Grpc.Core.Tests/project.json

```diff
@@ -14,10 +14,10 @@
         },
         "copyToOutput": {
           "mappings": {
-            "nativelibs/windows_x64/grpc_csharp_ext.dll": "../../../vsprojects/x64/Debug/grpc_csharp_ext.dll",
-            "nativelibs/windows_x86/grpc_csharp_ext.dll": "../../../vsprojects/Debug/grpc_csharp_ext.dll",
-            "nativelibs/linux_x64/libgrpc_csharp_ext.so": "../../../libs/dbg/libgrpc_csharp_ext.so",
-            "nativelibs/macosx_x64/libgrpc_csharp_ext.dylib": "../../../libs/dbg/libgrpc_csharp_ext.dylib"
+            "grpc_csharp_ext.x64.dll": "../../../vsprojects/x64/Debug/grpc_csharp_ext.dll",
```

Comment:
Hi Jon,

I would just open a `cmd.exe` window, and launch a Bash shell via Cygwin as follows:

```
c:\cygwin\bin\bash.exe --login -i
```

Then proceed to where your Bash script is located, and run it manually as on a normal Linux/Unix box.  The drives on Windows are located under `/cygdrive` like this:

```
$ ls /cygdrive
c
```

This way when running the script, you can see the debug information without having to worry that the window will close.

Hope it helps,
~p